ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp

```diff
@@ -190,6 +192,11 @@ uint32_t RSkTextLayoutManager::buildParagraph (SharedColor backGroundColor,
         style.setDecoration(fragment.textAttributes.textDecorationLineType.has_value() ?
                                 convertDecoration(fragment.textAttributes.textDecorationLineType.value()) :
                                 TextDecoration::kNoDecoration);
+        if(!backGroundColor) {
```

Comment:
Container Shadow opacity is handled in drawShadow API, but for text content shadow , opacity is not handled here. Please add TODO comment as text content shadow opacity to be handled.